nrich-validation/src/test/java/net/croz/nrich/validation/constraint/util/OibValidatorUtilTest.java

```diff
@@ -0,0 +1,19 @@
+package net.croz.nrich.validation.constraint.util;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class OibValidatorUtilTest {
+
+    @CsvSource({ "invalid,false", "111111,false", "98129812111,false", "22492583858,true" })
```

Comment:
Can you also add a test case where the [control number is 0](https://app.codecov.io/gh/croz-ltd/nrich/compare/104/tree/nrich-validation/src/main/java/net/croz/nrich/validation/constraint/util/OibValidatorUtil.java#D1L42), because there is some logic related to that in the validator? (control number initially 10, because of modulo 1...)